src/shared/install.h

```diff
@@ -217,8 +217,15 @@ typedef struct {
         bool initialized;
 } UnitFilePresets;
 
+typedef enum {
+        PRESET_UNKNOWN,
+        PRESET_ENABLE,
+        PRESET_DISABLE,
+        PRESET_IGNORE,
```

Comment:
please do the usual _PRESET_ACTION_MAX and _PRESET_ACTION_INVALID dance

Reply:
also do _PRESET_ACTION_ERRNO_MAX stuff since you want to return the full valid errno range in unit_file_query_preset()